src/screens/ChannelListScreen/elements/Search.js

```diff
@@ -26,14 +27,9 @@ const Search = ({onPress, animatedValue, isAnon, isShowNewChat = true}) => {
       <Pressable onPress={__handleOnSearchClicked} style={styles.searchPressableContainer}>
         <View style={styles.wrapperSearch}>
           <View style={styles.wrapperIcon}>
-            <MemoIc_search width={16.67} height={16.67} />
+            <MemoIc_search width={17} height={17} fill={COLORS.gray300} />
           </View>
-          <Text
-            // placeholder={StringConstant.chatTabHeaderPlaceholder}
-            // placeholderTextColor={COLORS.lightgrey}
-            style={styles.input}>
-            {StringConstant.chatTabHeaderPlaceholder}
-          </Text>
+          <Text style={styles.input}>{StringConstant.chatTabHeaderPlaceholder}</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The SVG icon size has been changed and the fill color has been added. Please ensure that these changes do not distort the icon or make it less visible. Also, the placeholder text color seems to have been removed. If this was intentional and the default color is suitable, then it's fine. Otherwise, consider adding it back.

```diff
- <MemoIc_search width={16.67} height={16.67} />
+ <MemoIc_search width={17} height={17} fill={COLORS.gray300} />
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->